integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/ESBIntegrationTest.java

```diff
@@ -889,6 +892,22 @@ protected void verifySequenceExistence(String sequenceName) throws RemoteExcepti
                 "Sequence not found. " + sequenceName);*/
     }
 
+    protected boolean isSequenceExists(String sequenceName) throws IOException {
+
+        String response = checkArtifactUsingManagementApi("sequences");
+        return response.contains(sequenceName);
+    }
+
+    private String checkArtifactUsingManagementApi(String artifactType) throws IOException {
+
+        SimpleHttpClient client = new SimpleHttpClient();
+        Map<String, String> headers = new HashMap<>();
+        headers.put("Accept", "application/json");
+
+        HttpResponse response = client.doGet("https://localhost:9354/management/" + artifactType, headers);
```

Comment:
@sajinieKavindya @Nirothipan 
Shall we not hard code the hostname and port?

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/466